base_layer/transaction_components/src/transaction_components/wallet_output.rs

```diff
@@ -426,6 +429,11 @@ impl WalletOutput {
         Ok(input)
     }
 
+    /// Get the deterministic TxId for this output
+    pub fn to_tx_id(&self) -> TxId {
```

Comment:
```suggestion
    pub fn calculate_tx_id(&self) -> TxId {
```
to implies conversion